scripts/exregional_novarcldanl.sh

```diff
@@ -0,0 +1,299 @@
+else                                               # Use background from RESTART
+  ln_vrfy -s ${bkpath}/fv_core.res.tile1.nc         fv3_dynvars
+  ln_vrfy -s ${bkpath}/fv_tracer.res.tile1.nc       fv3_tracer
+  ln_vrfy -s ${bkpath}/sfc_data.nc                  fv3_sfcdata
+fi
+
+#
+#-----------------------------------------------------------------------
+#
+# link/copy observation files to working directory
+#
+#-----------------------------------------------------------------------
+
+PROCESS_BUFR_PATH=${CYCLE_DIR}/process_bufr
+
+obs_files_source[0]=${PROCESS_BUFR_PATH}/LightningInFV3LAM.dat
+obs_files_target[0]=LightningInFV3LAM.dat
+
+obs_files_source[1]=${PROCESS_BUFR_PATH}/NASALaRC_cloud4fv3.bin
+obs_files_target[1]=NASALaRC_cloud4fv3.bin
+
+obs_files_source[2]=${PROCESS_BUFR_PATH}/fv3_metarcloud.bin
+obs_files_target[2]=fv3_metarcloud.bin
+
+obs_number=${#obs_files_source[@]}
+for (( i=0; i<${obs_number}; i++ ));
+do
+  obs_file=${obs_files_source[$i]}
+  obs_file_t=${obs_files_target[$i]}
+  if [ -r "${obs_file}" ]; then
+    cp_vrfy "${obs_file}" "${obs_file_t}"
+  else
+    print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+  fi
+done
+
+#-----------------------------------------------------------------------
+#
+# Build namelist 
+#
+#-----------------------------------------------------------------------
+
+cat << EOF > gsiparm.anl
+
+ &SETUP
+  iyear=${YYYY},
+  imonth=${MM},
+  iday=${DD},
+  ihour=${HH},
+  iminute=00,
+ /
+ &RAPIDREFRESH_CLDSURF
+   dfi_radar_latent_heat_time_period=20.0,
+   metar_impact_radius=10.0,
+   metar_impact_radius_lowCloud=4.0,
+   l_pw_hgt_adjust=.true.,
+   l_limit_pw_innov=.true.,
+   max_innov_pct=0.1,
+   l_cleanSnow_WarmTs=.true.,
+   r_cleanSnow_WarmTs_threshold=5.0,
+   l_conserve_thetaV=.true.,
+   i_conserve_thetaV_iternum=3,
+   l_cld_bld=.true.,
+   l_numconc=.true.,
+   cld_bld_hgt=1200.0,
+   build_cloud_frac_p=0.50,
+   clear_cloud_frac_p=0.10,
+   iclean_hydro_withRef_allcol=1,
+   i_gsdcldanal_type=6,
+   i_gsdsfc_uselist=1,
+   i_lightpcp=1,
+   i_gsdqc=2,
+   l_saturate_bkCloud=.true.,
+ /
+EOF
+
+
+
+#
+#-----------------------------------------------------------------------
+#
+# Copy the executable to the run directory.
+#
+#-----------------------------------------------------------------------
+#
+EXEC="${EXECDIR}/fv3sar_novarcldana.exe"
+
+if [ -f $EXEC ]; then
+  print_info_msg "$VERBOSE" "
+Copying the noVar Cloud Analysis executable to the run directory..."
+  cp_vrfy ${EXEC} ${workdir}
+else
+  print_err_msg_exit "\
+The executable specified in EXEC does not exist:
+  EXEC = \"$EXEC\"
+Build executable and rerun."
+fi
+#
+#
+#
+#-----------------------------------------------------------------------
+#
+# Run the novar cloud analysis application.  
+#
+#-----------------------------------------------------------------------
+#
+$APRUN ./fv3sar_novarcldana.exe > stdout 2>&1 || print_err_msg_exit "\
```

Comment:
There is duplication of information here. I'd suggest defining `EXEC=fv3sar_novarcldana.exe`, then in the `cp_vrfy` line, go with:

```cp_vrfy ${EXEC} ${workdir}```

So then here you can use:

```$APRUN $EXEC ...```

Reply:
Done